paperqa/agents/task.py

```diff
@@ -487,3 +493,248 @@ def __len__(self) -> int:
     LitQAv2TaskDataset.__module__,
     LitQAv2TaskDataset.__name__,
 )
+
+
+class LFRQAPairwiseEvalEnv(GradablePaperQAEnvironment):
+    def __init__(
+        self,
+        qid: str,
```

Comment:
If it's an ID, any chance we can do `UUID`, or at least `str | UUID`?

Reply:
Probably `str | UUID` because they come from the lfrqa dataset and are strings like `science-forum-test-240`